usr/share/rear/prep/default/310_include_uefi_tools.sh

```diff
@@ -8,7 +8,9 @@ if [[ ! -d /boot/[eE][fF][iI] ]]; then
     if is_true $USING_UEFI_BOOTLOADER; then
```

Comment:
Given that the whole block is executed when the directory does not exist, what is the point of checking whether it is a mount point?
Shouldn't it be rather
```sh
if [[ ! -d /boot/[eE][fF][iI] || ! mountpoint -q /boot/[eE][fF][iI] ]]; then
```
?

Reply:
@pcahyna , correct . I have edited the code

Reply:
Sorry this is not correct either, i will provide a better fix later today.